static/src/javascripts/projects/common/modules/commercial/epic/optimize-epic-utils.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import config from 'lib/config';
+import { getLocalCurrencySymbol } from 'lib/geolocation';
+import { constructQuery as constructURLQuery } from 'lib/url';
+
+import { displayIframeEpic } from 'common/modules/commercial/epic/iframe-epic-utils';
+
+import type { IframeEpicComponent } from 'common/modules/commercial/epic/iframe-epic-utils';
+
+const getOptimizeEpicUrl = (): string => {
+    const url =
+        config.get('page.optimizeEpicUrl') ||
+        // FIXME
+        'http://reader-revenue-components.s3-website-eu-west-1.amazonaws.com/epic/v1/index.html';
```

Comment:
you can pass this as a second arg to `config.get()`. But also, is it overkill to put a fallback serverside and clientside? and I guess the `FIXME` was to change this to the Fastly URL?